packages/frontend-core/src/api/user.js

```diff
@@ -15,7 +15,7 @@ export const buildUserEndpoints = API => ({
    */
   fetchBuilderSelf: async () => {
     return await API.get({
-      url: "/api/global/users/self",
+      url: "/api/global/self",
```

Comment:
We should probably move these endpoints into the new `self` file then shouldn't we? If we have a file for just the `self` routes then these 3 endpoints (`fetchSelf`, `fetchBuilderSelf` and `updateSelf`) probably belong in there rather than in users.